client/src/components/FileViewer/index.jsx

```diff
@@ -39,7 +40,7 @@ class FileViewer extends React.Component {
       <div className={'markdown'}>
         {
           this.state.fileLoaded &&
-          <ReactMarkdown className={'markdown-preview'} source={this.state.fileText} skipHtml />
+          <ReactMarkdown className={'markdown-preview'} source={this.state.fileText} skipHtml={skipHtmlMain} disallowedTypes={disallowedTypesMain} escapeHtml={escapeHtmlMain} />
```

Comment:
This change is **very** dangerous, can you make a list of what is **whitelisted**?

Blacklists are not safe when it comes to script injection.

Reply:
It is now a very long whitelist :)